update security (#1740)
emqtt/emq-docs-cn: en_US/deploy/cluster/security.md

# Cluster security

> EMQX provides several security mechanisms to ensure the confidentiality, integrity, and availability of data, including [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level, [a security cookie](#set-node-cookie) to ensure secure communication between nodes in a cluster, [TLS/SSL encryption](#configure-tls-ssl-to-secure-cluster-connections) to provide end-to-end encryption for inter-node traffic, as well as [port mapping rules](#port-mapping) to ensure that the firewall rules are configured correctly. 

Reviewer: ```suggestion
EMQX provides several security mechanisms to ensure the confidentiality, integrity, and availability of data, including [authentication](../../access-control/authn/authn.md) and [authorization](../../access-control/authz/authz.md) mechanisms on the node level, [a secret cookie](#set-node-cookie) to ensure secure communication between nodes in a cluster, [TLS/SSL encryption](#configure-tls-ssl-to-secure-cluster-connections) to provide end-to-end encryption for inter-node traffic.
```

> 

Reviewer: To help provisioning firewall rules, we will also touch on EMQX's inter-broker communication port mapping rules.
